debian/rules

```diff
@@ -149,7 +150,7 @@ override_dh_installinit:
 	dh_installinit -pcloudstack-management -pcloudstack-agent -pcloudstack-usage --onlyscripts --no-start
 
 override_dh_systemd_enable:
-	dh_systemd_enable -pcloudstack-agent -pcloudstack-usage
+	dh_systemd_enable -pcloudstack-management -pcloudstack-agent -pcloudstack-usage
```

Comment:
@wido we should if we do not support ubuntu 12.04 anymore. I prefer to remove it in 4.10. I can create another PR for it. This PR is only for 4.9 branch.